yarn.lock

```diff
@@ -2473,9 +2473,11 @@
     to-fast-properties "^2.0.0"
 
 "@backstage/catalog-model@^0.2.0":
-  version "0.7.0"
+  version "0.2.0"
+  resolved "https://registry.npmjs.org/@backstage/catalog-model/-/catalog-model-0.2.0.tgz#e3fe2a4ddeb6a9b6ec480c80cb2b9c39cb245576"
+  integrity sha512-Y1ocdRpBlxK/VrJQjHlQd0bgADECd1B2NRjwd8ss46ibT5hwLvMOfD80+Fa7oPLu0ktJrH4lq0pNIIJIml48zA==
```

Comment:
So sorry - these annoying changes are introduced frequently in many of the PRs by `yarn add` and we don't want to introduce them. Try rebasing with `master` branch, copying over [`yarn.lock`](https://github.com/backstage/backstage/blob/master/yarn.lock) and running `yarn` again.

Reply:
When I did that, there were no changes to yarn.lock, is that right?

Reply:
If you use the lower version of the testing library, then yep